openedx/features/calendar_sync/utils.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+import os
+
+import boto3
+
+LOGGER = logging.getLogger(__name__)
+
+
+def calendar_sync_initial_email_content(course_name):
+    subject = _("Stay on Track")
+    body = _("""
+             <div>Sticking to a schedule is the best way to ensure that you
+             successfully complete your self-paced course. This schedule of assignment due dates for {course}
+             will help you stay on track!</div>
```

Comment:
Maybe don't include the `<div>` stuff in the translatable string. Here and below.